doc/rtd/topics/analyze.rst

```diff
@@ -43,117 +43,121 @@ execution.
+        00.00000s (modules-final/config-scripts-per-instance)
+        00.00000s (modules-final/config-scripts-per-boot)
+        00.00000s (modules-final/config-puppet)
+        00.00000s (modules-final/config-power-state-change)
+        00.00000s (modules-final/config-mcollective)
+        00.00000s (modules-final/config-lxd)
+        00.00000s (modules-final/config-landscape)
+        00.00000s (modules-final/config-chef)
+        00.00000s (modules-config/config-snap_config)
+        00.00000s (modules-config/config-ntp)
+        00.00000s (modules-config/config-disable-ec2-metadata)
+        00.00000s (init-network/setup-datasource)
+
+    1 boot records analyzed
 
 Show
 ----
 
 The ``show`` action is similar to ``systemd-analyze critical-chain`` which
 prints a list of units, the time they started and how long they took.
-Cloud-init has four :ref:`boot stages<boot_stages>`, and within each stage a number of modules may
-run depending on configuration. ``cloudinit-analyze show`` will, for each boot,
-print this information and a summary of the total time.
+Cloud-init has four :ref:`boot stages<boot_stages>`, and within each stage a
+number of modules may run depending on configuration.
+``cloudinit-analyze show`` will, for each boot, print this information and a
+summary of the total time.
 
 The following is an abbreviated example of the show output:
 
 .. code-block:: shell-session
 
-  $ cloud-init analyze show
-  -- Boot Record 01 --
-  The total time elapsed since completing an event is printed after the "@" character.
-  The time the event takes is printed after the "+" character.
-
-  Starting stage: init-local
-  |``->no cache found @00.01700s +00.00200s
-  |`->found local data from DataSourceNoCloud @00.11000s +00.40300s
-  Finished stage: (init-local) 00.94200 seconds
-
-  Starting stage: init-network
-  |`->restored from cache with run check: DataSourceNoCloud [seed=/dev/sr0][dsmode=net] @04.79500s +00.02100s
-  |`->setting up datasource @04.88900s +00.00000s
-  |`->reading and applying user-data @04.90100s +00.00400s
-  |`->reading and applying vendor-data @04.90500s +00.00800s
-  |`->activating datasource @04.95200s +00.00400s
-  Finished stage: (init-network) 02.72100 seconds
-
-  Starting stage: modules-config
-  |`->config-snap ran successfully @15.43100s +00.00100s
-  ...
-  |`->config-runcmd ran successfully @16.22300s +00.00100s
-  |`->config-byobu ran successfully @16.23400s +00.00100s
-  Finished stage: (modules-config) 00.83500 seconds
-
-  Starting stage: modules-final
-  |`->config-snappy ran successfully @16.87400s +00.00200s
-  |`->config-package-update-upgrade-install ran successfully @16.87600s +00.00100s
-  ...
-  |`->config-final-message ran successfully @16.93700s +00.00500s
-  |`->config-power-state-change ran successfully @16.94300s +00.00000s
-  Finished stage: (modules-final) 00.10300 seconds
-
-  Total Time: 4.60100 seconds
-
-  1 boot records analyzed
+    $ cloud-init analyze show
```

Comment:
All other subcommands in this section have separate example output and command. For consistency could we do the same here?

Reply:
Tox had a bit of a problem with this block specifically - when I tried to break up the block it started complaining about the lines inside it being too long - even though when they were part of one block, it happily accepted them (also, they weren't even too long). I can make the change, though.

Reply:
> when I tried to break up the block it started complaining about the lines inside it being too long - even though when they were part of one block, it happily accepted them 

Hrm, that's a bit annoying. Doing a thing that seemingly shouldn't break anything that causes breakage is never fun.

> (also, they weren't even too long)

This sounds like potentially a report-able bug. If it's still doing that with this version I can take a look at the specific warning and check if we should report to whichever project is causing the inaccurate warning.